// vsp-bot/index.js
const biodata = {
  name: "VSP Bot",
  createdOn: "28 April 2023",
  purpose:
    "VSP Bot helps you to know the details of King VSP (Vishnu Suresh Perumbavoor)",
  creator: "Vishnu Suresh Perumbavoor",
  whoAmI: ["engineer", "entrepreneur", "creator"],
  interests: [
    "startups",
    "flirting",
    "geopolitics",
    "history",
    "penny stocks and crypto",
    "forex market",
  ],
  entertainments: ["pubs", "IPL", "movies"],
  founderOf: "VSP Enterprises",
  sector: "edTech",
  motive: [
    "helps students to learn MERN stack, NextJS, Git, Docker, AI, ML, Blockchain, etc",
    "prepare students for hackthons, freelancing and interviews",
    "helps students to build interesting projects",
  ],
  goals: "Create DevOps and RPA engineers",
  futureplans: [
    "end the anthropocene epoch",
    "build metro cities across the planet",
    "make humanity a multiplanetary species",
    "make human civilization Type 1.0",
    "manipulate penny stocks and crypto currencies",
  ],
  internships: {
    1: "Infinio Technology Solutions",
    2: "Synergy Systems",
  },
  placement: "Experion Technologies",
  position: "Associate Software Engineer",
  accomplishments: [
    "Won 3rd prize in Vaiga Agrihack 2023, which is the biggest agricultual hackathon in Kerala",
    "Participated in Rajasthan IT Hackathon 2023",
    "Won 1st prize in startup idea presentation at Palakkad organized by IEDC, KSUM & IIT Palakkad",
    "Won 2nd prize in startup idea presentation at Kochi, MES AIMAT Marampilly, Aluva",
  ],
  socials: {
    twitter: "https://twitter.com/vspeeeeee",
    linkedin: "https://www.linkedin.com/in/vishnu-suresh-perumbavoor/",
    github: "https://github.com/vishnusureshperumbavoor",
    youtube: "https://www.youtube.com/@vishnusureshperumbavoor9721/videos",
    instagram: "https://www.instagram.com/vishnusureshperumbavoor/",
  },
  featured: {
    deshabhimani: "https://epaper.deshabhimani.com/c/72302051",
    newsAtNet: "https://www.newsatnet.com/news/local-news/144328/",
  },
};

module.exports = biodata;
