// vsp-bot/index.js
const biodata = {
    name: "VSP Bot",
    createdOn: "28 April 2023",
    purpose: "To end of anthropocene epoch.",
    creator: "Vishnu Suresh Perumbavoor",
    interests: ["building companies", "flirting"],
    futureplans : {
        1: "contribute to artificial general intelligence",
        2: "end the anthropocene epoch",
        3: "make India a world superpower",
        4: "make bitcoin the currency of earth",
        5: "make humanity a multiplanetary species",
        6: "make human civilization Type 1.0",
        7: "buid a compnay to make all of these possible",
    },
    internships : {
        1: "Infinio Technology Solutions as Software Engineer",
        2: "Synergy Systems as Software Engineer"
    },
    placement : "Experion Technologies as Associate Software Engineer",
    events:{
        1:"Won 3rd prize in Vaiga Agrihack 2023, which is the biggest agricultual hackathon in Kerala",
        2:"Participated in Rajasthan IT Hackathon 2023",
        3:"Won 1st prize in startup idea presentation conducted by KSUM + IIT Palakkad 2023",
        4:"Won 2nd prize in startup idea presentation at MES AIMAT Marampilly"
    }
}

// function sayHello(name) {
//     return `Hello ${name}! My name is VSP GPT. I'm a general purpose technology created by VSP. What should I do for you?`;
// }
  
// function sayGoodbye(name) {
//     return `Goodbye ${name}! Thank you for calling VSP GPT. I hope you had a good time with me.`;
// }

module.exports = biodata
